### PR TITLE
feat/SPRFRML-126: null safety migration

### PR DESCRIPTION
**Pull request by alberto-sf**
## Why?

We need to migrate this package to null safety in order to make our Iris project "sound"

## What?

I have run `dart migrate` and made some manual fixes to migrate this package

## Related Tickets

[SPRFRML-126](https://jira.alkami.com/browse/SPRFRML-126)

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -10,9 +10,10 @@ void main() {
 
   dashbook.storiesOf('RaisedButton').decorator(CenterDecorator()).add(
         'default',
-        (ctx) => RaisedButton(
+        (ctx) => ElevatedButton(
           child: Text(
-            ctx.listProperty("Label", "Ok", ["Ok", "Cancel", "Other label"]),
+            ctx.listProperty("Label", "Ok", ["Ok", "Cancel", "Other label"]) ??
+                "",
```
Comment (jeffs-sf) [inline example/lib/main.dart:16]: **question:** why does it need the default? shouldn't this not be null?

Comment (alberto-sf) [reply example/lib/main.dart:16]: Nope! The `listProperty` method returns `T?` and there you need a non-nullable value 😄

File: pubspec.yaml [hunk 2]
```diff
@@ -1,16 +1,17 @@
 name: dashbook
 description: A showcase/sandbox library to help you build isolated widgets in Flutter
-version: 0.0.11
+version: 1.0.0
```
Comment (jeffs-sf) [inline pubspec.yaml:3]: **thought:** this should be fine but we may end up going to `0.1.0` when the library bumps.

**Review by jeffs-sf — APPROVED** [review]
+1

**Pull request merged**